R/get_delay_estimate.R

```diff
@@ -1,57 +1,58 @@
 #' Estimate a delay distribution from a reporting triangle
 #' @description
 #' Provides an estimate of the reporting delay as a function
-#' of the delay, based on the reporting triangle and the specified maximum
-#' delay and number of reference date observations to be used in the estimation.
-#' This point estimate of the delay is computed empirically, using an
-#' iterative algorithm starting from the most recent observations. It was
-#' modified from the code originally developed by the Karlsruhe Institute
-#' of Technology RESPINOW German Hospitalization Nowcasting Hub,
-#' Modified from: https://github.com/KITmetricslab/RESPINOW-Hub/blob/7cce3ae2728116e8c8cc0e4ab29074462c24650e/code/baseline/functions.R#L55 #nolint
-#' @param triangle matrix of the reporting triangle, with rows representing
-#' the time points of reference and columns representing the delays
-#' indexed at 0
-#' @param max_delay integer indicating the maximum delay to estimate, in units
-#' of the delay. The default is to use the whole reporting triangle,
+#'  of the delay, based on the reporting triangle and the specified maximum
+#'  delay and number of reference date observations to be used in the
+#'  estimation. This point estimate of the delay is computed empirically,
+#'  using an iterative algorithm starting from the most recent observations.
+#'  This code was adapted from code written (under an MIT license)
+#'  by the Karlsruhe Institute of Technology RESPINOW
+#'  German Hospitalization Nowcasting Hub.
+#'  Modified from: https://github.com/KITmetricslab/RESPINOW-Hub/blob/7cce3ae2728116e8c8cc0e4ab29074462c24650e/code/baseline/functions.R#L55 #nolint
+#' @param triangle Matrix of the reporting triangle, with rows representing
+#'  the time points of reference and columns representing the delays
+#' @param max_delay Integer indicating the maximum delay to estimate, in units
+#'  of the delay. The default is to use the whole reporting triangle,
```

Comment:
Im just here being that guy saying this only has one space and the other docs have two... the only thing that matters is they have the same number to me

Reply:
fixing